charts/collab/README.md

```diff
@@ -0,0 +1,113 @@
+| `av.stunServers.enabled`                    | Enables private STUN servers                                                                  | `false`                                                   |
+| `av.stunServers.server`                     | STUN server address                                                                           | `coturn.example.com`                                      |
+| `av.stunServers.secret`                     | STUN server secret                                                                            | `my-secret`                                               |
+| `sso.enabled`                               | Enables Single Sign-On (SSO) integration                                                      | `false`                                                   |
+| `sso.provider`                              | SSO provider name (e.g., OKTA, MICROSOFT, KEYCLOAK)                                           | `OKTA`                                                    |
+| `sso.systemBaseUrl`                         | Base URL for the identity provider                                                            | `https://OKTA_DOMAIN/oauth2/default`                      |
+| `sso.clientID`                              | Client ID for the SSO provider                                                                | `my-oidc-id`                                              |
+| `sso.clientSecret`                          | Client secret for the SSO provider                                                            | `my-id-secret`                                            |
+| `service.type`                              | Kubernetes service type                                                                       | `ClusterIP`                                               |
+| `service.port`                              | Port for CodeTogether Collab service                                                         | `443`                                                     |
+| `restart.enabled`                           | Enables periodic restarts for the server                                                     | `true`                                                    |
+| `restart.cronPattern`                       | Cron pattern for scheduling restarts                                                         | `* 11 * * 0`                                              |
+| `favicon.enabled`                           | Enables a custom favicon                                                                     | `false`                                                   |
+| `favicon.filePath`                          | Path to the custom favicon                                                                   | `files/new-favicon.ico`                                   |
+
+## Creating your Kubernetes Namespace for CodeTogether Collab
```

Comment:
I would also add a command example on how to create the tls secret

codetogether-tls